.changeset/fluffy-steaks-flow.md

```diff
@@ -0,0 +1,10 @@
+---
+"@logto/schemas": minor
+"@logto/core": minor
+"@logto/console": minor
+---
+
+refactor the definition of hook event types:
```

Comment:
the first line is a **preview** of the change so it's not appropriate for an ending colon:

```suggestion
refactor the definition of hook event types
```